### PR TITLE
(DOCSP-44132): Admin API: Triggers & Endpoints - provide info on getting the service ID

### PR DESCRIPTION
**Pull request by dacharyc**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-44132

### Staging Links
Staging links are not available for the App Services Admin API.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [ ] Did you tag pages appropriately?
  - genre
  - programming_language
  - meta.keywords
  - meta.description
- [ ] Describe your PR's changes in the Release Notes section
- [ ] Create a Jira ticket for related docs-realm work, if any

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/openapi-admin-v3.yaml [hunk 1]
```diff
@@ -5547,10 +5547,14 @@ components:
             mongo_service_id:
               type: string
               description: |-
-                The `_id` value of the linked MongoDB data source that contains
-                custom user data.
-
-                See [Get a Data Source](#operation/adminGetService).
+                The _id value of a linked MongoDB data source.
+                    
+                You can list services with the the
```
Comment (cbullinger) [inline source/openapi-admin-v3.yaml:5552]:
```suggestion
                You can list services with the
```

File: source/openapi-admin-v3.yaml [hunk 2]
```diff
@@ -7847,8 +7851,13 @@ components:
                   type: string
                   description: |
                     The _id value of a linked MongoDB data source.
+                    
+                    You can list services with the the
```
Comment (cbullinger) [inline source/openapi-admin-v3.yaml:7855]:
```suggestion
                    You can list services with the
```

**Review by cbullinger — APPROVED** [review]
😎

File: source/openapi-admin-v3.yaml (hunk 1 above)
Comment (dacharyc) [reply source/openapi-admin-v3.yaml:5552]: Oof, thank you for catching this! 😅

**Pull request merged**